JMS/src/test/java/com/ibm/mq/samples/jms/JmsPutTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+* (c) Copyright IBM Corporation 2024
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.ibm.mq.samples.jms;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import javax.jms.JMSConsumer;
+import javax.jms.JMSContext;
+import javax.jms.JMSException;
+import javax.jms.Destination;
+import javax.jms.Message;
+import javax.jms.TextMessage;
+import javax.jms.JMSRuntimeException;
+
+import org.junit.jupiter.api.Test;
+
+import com.ibm.mq.MQException;
+import com.ibm.mq.constants.MQConstants;
+
+
+public class JmsPutTest {
+    private JMSContext context = null;
+    private Destination destination = null;
+    private JMSConsumer consumer = null;
+    private ConnectionHelper ch = null;
+    private static long TIMEOUTTIME = 5000;
+    @Test
+    public void testJmsPut(){
+        clearQueue();
```

Comment:
add comment - clears unto 1000 messages of the queue. If there are more then this test will fail.